app/build.gradle.kts

```diff
@@ -58,6 +59,9 @@ android {
 
     buildFeatures.viewBinding = true
     kapt.correctErrorTypes = true
+    lintOptions {
+        disable("InvalidFragmentVersionForActivityResult")
```

Comment:
This is a legitimate lint pattern and almost never a good idea to suppress.

Reply:
I have updated my `fragments-ktx` to `1.3.0-alpha08` but it was still showing this lint error so I'm guessing it's a false positive.

Reply:
Try beta01

Reply:
Did not help :(

Reply:
Weird, I did the update in WireGuard and it worked https://github.com/WireGuard/wireguard-android/commit/9266487fe572e939170d58ab2a5c2a0e138ffbc1

Reply:
Yeah the problem is not that I'm not following the pattern but `ActivityResultFragmentVersionDetector.checkDslPropertyAssignment` accepts a parent parameter of type `String` but it is being passed as `null`.